extension/runtime/src/main/java/io/quarkiverse/temporal/TemporalWorkflowStub.java

```diff
@@ -17,24 +18,54 @@
 @Retention(RUNTIME)
 @Target({ METHOD, FIELD, PARAMETER, TYPE })
 public @interface TemporalWorkflowStub {
+
     String DEFAULT_WORKER = "io.quarkiverse.temporal.TemporalWorkflowStub.worker.DEFAULT";
 
     String DEFAULT_WORKFLOW_ID = "io.quarkiverse.temporal.TemporalWorkflowStub.workflowId.DEFAULT";
 
+    /**
+     * This allows to customize the workflow stub configuration.
+     * Letting this blank will bind to the unnamed configuration group.
+     */
+    String group() default DEFAULT_WORKFLOW_GROUP_NAME;
```

Comment:
is this the right name ? not sure it is explicit enough

Reply:
i feel like group is OK. Not sure what else you would call it?

Reply:
yeah I don't really have a better idea